### DISCUSSION TITLE
我已经绑定了高级账户，但是打开稍后读后有些地方仍是被锁状态，在与同步助手链接的时候也是灰色（解决方案）

### DISCUSSION THREAD

**Discussion by castlelong** [Kenshin/simpread#2816] [高级账户]
![image](https://user-images.githubusercontent.com/26018270/113807488-bfbf5800-9753-11eb-8a38-3292479990ae.png)
![image](https://user-images.githubusercontent.com/26018270/113807509-c9e15680-9753-11eb-8092-a8c00c03f5c9.png)

**Comment by Kenshin** [comment]
## 前提

> 首先确保此选项已开启 https://github.com/Kenshin/simpread/discussions/1567 引起的？

## 解决方案

1. 解除当前设备的授权

> 选项页 → 右上角 → 👤  如果跟下图一致的话，请点击 **解除当前设备的授权** （否则请忽略此步骤）

![image](https://user-images.githubusercontent.com/81074/132818165-4491304d-3a6b-4b0b-abd0-067e5f8abbca.png)

2. 重启浏览器

3. 选项页 → 右上角 → 👤 → **仅授权当前设备**

![image](https://user-images.githubusercontent.com/81074/132818269-ee905779-4724-47aa-863a-e31e6394b876.png)

4. 稍等片刻后即可成功解决此问题。

**Reply by Wonz5130** [reply]
Hi, K大。如果之前的电脑忘记解除当前设备的授权，新电脑如何保证单独使用高级帐户？

例如：我买了一台新的Macbook进行数据同步之后，新电脑上已经显示是高级帐户，但是每次使用简悦做标注都会弹出以下信息：

**Reply by Kenshin** [reply]
> 如果之前的电脑忘记解除当前设备的授权，新电脑如何保证单独使用高级帐户？

14 天之后旧电脑的高级账户自动删除。（如果不再登录的话）

> 新电脑上已经显示是高级帐户，但是每次使用简悦做标注都会弹出以下信息：

安装 https://github.com/Kenshin/simpread/discussions/2816 设置就好了，或者看下同步助手（如果用的话），是不是没有绑定？如果是的话，需要重新绑下  https://www.yuque.com/kenshin/simpread/pwpnsx
